### DISCUSSION TITLE
4.1.4 Notices (1) There is an update available for your system. Dashboard states "This website is running latest Chevereto version"

### DISCUSSION THREAD

**Discussion by scottrbaxter** [chevereto/chevereto#83] [Q&A]
It looks as though Version 4.2 was released a few days ago, and a deploy via cpanel continues to alert that there is an update, however the dashboard explicitly states it is running the latest version, which is 4.1.4. I see no clear way forward to 4.2.

I also do not see 4.2 while logged through the [panel](https://chevereto.com/panel), either. Additionally, I wanted to test out the new release and attempted to build a local container, however `chevereto.sh` script seemingly fails to download 4.2. It seems as though the url is broken.

```
* NOTE: Using [[ PAID ]] edition package
* Downloading Chevereto V4.2 package
> https://chevereto.com/api/download/4.2
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 400
```

I've looked through the blog, forums, discord and repo issues with no clear indicator... was version 4.2 pulled temporarily? At most, I see that [81](https://github.com/chevereto/chevereto/issues/81) is related to user issues after upgrading. Aside from some troubleshooting steps for upgrades, there's no mention of the current status of version 4.2. As of today, I cannot download and manually install 4.2 through cpanel, or build a 4.2 container through the recommended [pure-docker](https://github.com/chevereto/docker/blob/4.2/docs/PURE-DOCKER.md) solution.

What is the recommended way forward?

**Comment by rodber** [comment]
> there's no mention of the current status of version 4.2

What about this: https://chevereto.com/community/threads/distribution-of-v4-2-0-is-paused.15661
